### DISCUSSION TITLE
No funciona yi hack - no lee firmware

### DISCUSSION THREAD

**Discussion by PastelBeltran** [roleoroleo/yi-hack-Allwinner#381] [Q&A]
Hola, compré hace un tiempo 4 cámaras home con el chip Y20ga mb_v1.3 2019/12/09. Las tuve apagadas un tiempo, quiero instalarles yi hack pero no reconoce nada. Ya probé todos los firmwares posibles y no permite. Son de la serie BFUS. En algún foro leí que habían bloqueado la posibilidad de cambiarle el firmware? existe alguna alternativa? quizás alguien ya lo modifico? pls toda info es buena, llevo tiempo buscando.

**Comment by roleoroleo** [comment]
Please check the fw version inside the app.
The model suffix depends on the fw version (or vice versa).
And I don't think they blocked the ability to change the firmware.

**Reply by PastelBeltran** [reply]
Hola, este es el firmware: 8.2.0.0A_201912270941 . 

En algún momento leí entre los distintos foros que otras cámaras de determinados años tenían el mismo problema. Estas cámaras fueron compradas hace un tiempo pero nunca pude instalarles yi hack. Alguna idea? qué puedo intentar? Ayuda pls

**Reply by roleoroleo** [reply]
it is exactly the same camera I have.
Please, check again the install procedure.
